ts/components/ui/RNavScreenWithLargeHeader.tsx

```diff
@@ -22,11 +22,16 @@ import {
 import { SupportRequestParams } from "../../hooks/useStartSupportRequest";
 import I18n from "../../i18n";
 
+export type LargeHeaderTitleProps = {
+  label: string;
+  accessibilityLabel: string;
+  testID?: string;
+};
+
 type Props = {
   children: React.ReactNode;
   fixedBottomSlot?: React.ReactNode;
-  title: string;
-  titleTestID?: string;
+  title: LargeHeaderTitleProps;
```

Comment:
```suggestion
  titleProps: LargeHeaderTitleProps;
```
since fe are using this naming convention for the HeaderActions, I think this brings more consistency; what do you think?

Reply:
also, are you sure we want to force the a11y label instead of inferring it from the label if undef?

Reply:
About the renaming from `title` to `titleProps` I don't like it very much as a naming convention, indeed I'm following the naming convention also used on other components inside the design system library.

About the `accessibilityLabel` as required, I agree with you and as a follow-up to today's meeting, I changed it as an optional property: 314de26009e68524a520a51e407f77a95823fe42